nexus/analyzer/src/lib.rs

```diff
@@ -800,12 +800,16 @@ fn parse_db_options(
     Ok(config)
 }
 
-fn parse_metadata_db_info(conn_str: Option<&str>) -> anyhow::Result<Option<PostgresConfig>> {
+fn parse_metadata_db_info(conn_str: Option<&&str>) -> anyhow::Result<Option<PostgresConfig>> {
```

Comment:
Caller can change `Option<&&str>` to `Option<&str>` with `.copied()`

Reply:
is it better than taking in `Option<&str>` ?

Reply:
Yes, because `&&str` isn't really what you want, it's just a side effect of HashMap returning a reference to its `&str` value

Reply:
done!